app/api/candidate/route.ts

```
@@ -0,0 +1,164 @@
+      city,
+      state,
+      bio,
+      website: website || null,
+      linkedin: linkedin || null,
+      policies,
+      clerkUserId,
+      additionalNotes: additionalNotes || null,
+    };
+
+    // Explicitly tell Prisma to use its own ID generation
+    delete createData.id; // Make sure no ID is being passed
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)